src/main/resources/db/changeLog.xml

```diff
@@ -380,4 +380,12 @@
             </column>
         </addColumn>
     </changeSet>
+
+    <changeSet id="dependent user to task" author="Lauri Lukkarinen">
+        <addColumn tableName="task">
+            <column name="dependentUser_id" type="binary(16)">
```

Comment:
Should the column be lower case e.g. `dependentuser_id`?

Reply:
Changed.